6-Typescript/server/middlewares/checkCookie.ts

```diff
@@ -1,13 +1,12 @@
+import { v4 as uuid } from "uuid";
+import { RequestHandler } from "express";
 
-import { v4 as uuid } from 'uuid';
-import {Request, Response, NextFunction} from 'express'
-
-export const checkCookie = function (req: Request, res: Response, next: NextFunction) {
-    const userId: string = req.cookies['userId'];
+export const checkCookie: RequestHandler = function (req, res, next) {
```

Comment:
another just a lint fix.. hard to follow :) 
nothing to fix in this pr - but note for the next time